cli/operator/register.go

```diff
@@ -136,21 +136,38 @@ func RegisterCmd(p prompter.Prompter) *cobra.Command {
 				llog,
 				noopMetrics,
 			)
+			if err != nil {
+				return err
+			}
 
+			reader, err := elContracts.NewELChainReader(
```

Comment:
Why aren't you using the constructor we built?

Reply:
yea. I wanted to - I need to overhaul this whole interaction with constructor - I didn't because we were using slasher contract address but constructor is expecting other one (state operator something) so I will update it later